src/starpoint.c

```diff
@@ -0,0 +1,151 @@
+#include "common.h"
+#include "entity.h"
+
+Lights1 starpoint_lights1 = gdSPDefLights1(0, 0, 0, 0, 0, 0, 0, 0, 0);
+#include "starpoint/starpoint.png.inc.c"
+#include "starpoint/load_starpoint.gfx.inc.c"
+#include "starpoint/starpoint.vtx.inc.c"
+Gfx D_802A20D0_7A9A70[] =
+{
```

Comment:
Extend this change to all structs here please
```suggestion
Gfx D_802A20D0_7A9A70[] = {
```